packages/i18n/src/locales/en/components/workbench.yaml

```diff
@@ -5,3 +5,4 @@ layoutThing: 'Layout {thing}'
 pageSize: Page size
 startBySelectingAThing: 'Start by selecting a {thing}'
 testThing: 'Test {thing}'
+editThingTitle: 'Edit Pattern Configuration for {thing}'
```

Comment:
Whenever I use `thing` it's typically because it allows replacing `thing` with something else. Like Edit `username`, or Edit `pattern`, or Edit `settings`, or edit `newsletter preferences`.

But there we're setting `editThingTitle` to something specific to the pattern configuration.
I'm afraid that would be counter-intuitive in those other scenarios where `editThing` is used.

I appreciate that managing translation keys across the project is not so obvious. But I'd like to see a more descriptive key here. Like `editPatternConfigThingTitle`.

Reply:
How do we feel about this?

```yaml
editView:
  titleThing: 'Edit Pattern Configuration for {thing}'
  error: Issues with YAML
  errorDesc: We saved your input, but it might not work for the following reasons
```

Reply:
We juggle these yaml files in various places, and we expect a flat key/value structure.
This would break that, so please let's not.

Can we not just do something like:

- `patternEditViewTitle`
- `patternEditViewError`
- `patternEditViewErrorDesc`

?

Reply:
For sure, sorry. I hate coming up with translation keys

Reply:
I went with `yamlEditView` etc, and I did leave it as `yamlEditViewTitleThing` because it has a thing in it. Hope this is acceptable!

Reply:
@joostdecock should I take this still being here as a sign that these are still not acceptable, or just that it got buried?